infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -23,6 +23,60 @@ nightly_builds = [
 
 # Built on push to specific tag.
 versioned_builds = [
+  # Remove libtpu from PyPI builds
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    bundle_libtpu   = "0"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    python_version  = "3.9"
+    bundle_libtpu   = "0"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    python_version  = "3.10"
+    bundle_libtpu   = "0"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    python_version  = "3.11"
+    bundle_libtpu   = "0"
+  },
+  # Bundle libtpu for Kaggle
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1+libtpu"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    python_version  = "3.10"
+    bundle_libtpu   = "1"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    accelerator     = "cuda"
+    cuda_version    = "12.1"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    accelerator     = "cuda"
+    cuda_version    = "12.1"
```

Comment:
Can you double check that this is the CUDA version upstream is using for their release?

Reply:
Do you know how to check this cc @vanbasten23 as well.

Reply:
Asking the PyTorch team offline, add more triggers for CUDA if necessary.